### DISCUSSION TITLE
Blaue Menü

### DISCUSSION THREAD

**Discussion by Geso1003** [tbnobody/OpenDTU#2105] [Q&A]
Habe eine Duo DTU.Darauf sind zwei Wechselrichter.HM800 und HMS1600.
Der HM 800 hat ein blaues Menü und der HMS 1600 zeigt ein rotes Menü an.
Was muss ich machen?

**Comment by Marc9485** [comment]
Blau = alles i.O.
Rot = keine Verbindung zum WR
